resources/dicts/patrols/forest/med/greenleaf.json

```diff
@@ -2463,5 +2463,37 @@
         "max_cats": 6,
         "antagonize_text": null,
         "antagonize_fail_text": null
+    },
+        {
+        "patrol_id": "fst_med_corpselily1",
+        "biome": "forest",
+        "season": "greenleaf",
+        "tags": [
+            "herb_gathering",
+            "med_cat",
+            "med_only",
+            "death",
+            "injury",
+            "poisoned",
+        ],
+        "intro_text": "p_l sets off to find the bulbs of a corpse lily. They work wonders for memory loss and diseases of the mind, but they come at a high price.",
+        "decline_text": "Still, they're lilies - poisonous and deadly. p_l decides to not risk {PRONOUN/p_l/poss} life.",
+        "chance_of_success": 55,
+        "exp": 20,
+       "success_text": {
+            "unscathed_common": "p_l finds a cluster of corpse lilies, wilting. The flowers, red and spindly, droop towards the floor. A few of the bulbs will be salvagable, if wrapped tightly in cobwebs and leaves for protection. Fortunately, there's plenty of cobwebs and enough tree leaves to wrap the bulb entirely.",
```

Comment:
We don't currently have lilies as a collective herb, so it might be confusing for the cat to "gather" them without any new herb added to the collection.